cmd/ackdev/cmd/config_view.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+
+	"github.com/aws-controllers-k8s/dev-tools/pkg/config"
+)
+
+var (
+	optViewConfigOutput string
+)
+
+func init() {
+	viewConfigCmd.PersistentFlags().StringVarP(&optViewConfigOutput, "output", "o", "yaml", "output format (json|yaml)")
```

Comment:
And I would add this arg to https://github.com/aws-controllers-k8s/dev-tools/blob/3187ceb3b3d6551405fa33d57a75b86ab11b71e5/cmd/ackdev/cmd/list.go#L22 instead.

Reply:
Very good point! Moved `--ouput` flag to the list command.